src/Elastic.Apm/Helpers/ConfigHelper.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Configuration;
+using System.Text;
+using Elastic.Apm.Config;
+using Elastic.Apm.Logging;
+
+namespace Elastic.Apm.Helpers
+{
+	public class ConfigHelper
+	{
+		public static IConfigurationReader CreateReader(IApmLogger logger)
+		{
+			if (ConfigurationManager.AppSettings[ConfigConsts.KeyNames.ConfigurationReaderType] != null
```

Comment:
It can be simplified to 

```csharp
if (!string.IsNullOrEmpty(ConfigurationManager.AppSettings[ConfigConsts.KeyNames.ConfigurationReaderType]))
```

Also, I suggest using `IsNullOrWhiteSpace`. And, maybe it's worth logging some warning message when the value is empty or whitespace.

Reply:
it was a mistake to let the !=null as i added the IsNullOrEmpty after..